Source/Server/Packets/KillAction.c

```diff
@@ -17,6 +17,11 @@ void send_kill_action_packet(server_t* server,
     if (player->alive == 0) {
         return; // Cant kill player if they are dead
     }
+
+    if (player->has_intel) {
+        send_intel_drop(server, player);
```

Comment:
Note: you need to send the intel drop packet BEFORE the kill packet. Pyspades does the same and I don't remember the reasons for this but it makes sense logically.